### PR TITLE
Add parcels feedstock

### PR DESCRIPTION
**Pull request by willirath**

### CODE REVIEW

File: recipes/parcels/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  description: |
+    Parcels (Probably A Really Computationally Efficient Lagrangian Simulator)
+    is an experimental prototype framework aimed at exploring novel approaches
+    for Lagrangian tracking of virtual ocean particles in the petascale age.
+  doc_url: http://oceanparcels.org/
+  dev_url: https://github.com/OceanParcels/parcels
+
+extra:
+  recipe-maintainers:
+    - erikvansebille
```
Comment (CJ-Wright) [inline recipes/parcels/meta.yaml]: @erikvansebille are you okay with being a maintainer?

File: recipes/parcels/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    - delandmeterp
```
Comment (CJ-Wright) [inline recipes/parcels/meta.yaml]: @delandmeterp are you ok with being a maintainer?

File: recipes/parcels/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    - progressbar2
+    - python-dateutil
+    - scipy >=0.16.0
+    - six >=1.10.0
+    - xarray >=0.5.1
+test:
+  imports:
+    - parcels
+  requires:
+    - pytest
```
Comment (CJ-Wright) [inline recipes/parcels/meta.yaml]: Why does an import test require pytest?

File: recipes/parcels/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    - netcdf4 >=1.1.9
+    - numpy >=1.9.1
+    - progressbar2
+    - python-dateutil
+    - scipy >=0.16.0
+    - six >=1.10.0
+    - xarray >=0.5.1
+test:
+  imports:
+    - parcels
```
Comment (CJ-Wright) [inline recipes/parcels/meta.yaml]: Would it be possible to add more import tests?

File: recipes/parcels/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    - m2w64-toolchain  # [win]
+    - matplotlib 2.0.2
+    - netcdf4 >=1.1.9
+    - numpy >=1.9.1
+    - progressbar2
+    - python-dateutil
+    - scipy >=0.16.0
+    - six >=1.10.0
+    - xarray >=0.5.1
+test:
```
Comment (CJ-Wright) [inline recipes/parcels/meta.yaml]: Could you please put a newline between the requirements and test sections?

File: recipes/parcels/meta.yaml (hunk 3 above)
Comment (willirath) [reply recipes/parcels/meta.yaml]: It doesn't. I falsely assumed that there was some kind of autodiscovery for actual unit tests.  I'll remove the requirement.

File: recipes/parcels/meta.yaml (hunk 4 above)
Comment (willirath) [reply recipes/parcels/meta.yaml]: Yes.  I'll add explicit imports of the relevant sub-modules.

File: recipes/parcels/meta.yaml (hunk 5 above)
Comment (willirath) [reply recipes/parcels/meta.yaml]: Will do.  Thanks for the review!

File: recipes/parcels/meta.yaml (hunk 1 above)
Comment (erikvansebille) [reply recipes/parcels/meta.yaml]: Yes, I'm ok with being a maintainer

File: recipes/parcels/meta.yaml (hunk 2 above)
Comment (delandmeterp) [reply recipes/parcels/meta.yaml]: Yes sure!

**Comment by willirath** [comment]
@CJ-Wright Thanks for the review.

Following suggestions made in #4803 I've skipped OSX builds for now in favor of getting the feedstock for at leas Linux and Windows (planning to re-enable OSX in the feedstock as soon as Travis recovers).

Are we good to go?

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes can someone with more C experience take a look? There seems to be one C header in the package and I don't know if that means it needs `toolchain` in the build

**Comment by willirath** [comment]
Great, thanks!


Am 24. Januar 2018 17:26:51 schrieb "Christopher J. Wright" <notifications@github.com>:

**Pull request merged**